### PR TITLE
Restarts with zeta flags

### PR DESCRIPTION
**Pull request by sjboeing**
Depends on sgs diagnostics

### CODE REVIEW

**Comment by sjboeing** [comment]
Leaving the lack of parameters as a warning for now, but let me know what you think.

File: src/3d/utils/parameters.f90 [hunk 1]
```diff
@@ -186,6 +186,8 @@ subroutine read_zeta_boundary_flag(ncid)
         else
             print *, "WARNING: Could not find a '" // name // "' group in the provided"
             print *, "         NetCDF file."
+            print *, "Note this will result in the boundary zeta flags being set to false when"
```
Comment (matt-frey) [inline src/3d/utils/parameters.f90:189]: You may want to indent this to make it clear it belongs to the warning.

**Review by matt-frey — COMMENTED** [review]
Yes, that is fine with me.

File: src/3d/utils/parameters.f90 (hunk 1 above)
Comment (sjboeing) [reply src/3d/utils/parameters.f90]: Thanks @matt-frey just made this change.

**Review by matt-frey — APPROVED** [review]
Thanks.

**Pull request merged**